cmd/keys_test.go

```diff
@@ -40,6 +40,9 @@ func storeOldKey(ts *TestStore, operator string, account string, user string, se
 	// <op>/accounts/<actname>/<actname>.nk
 	// <op>/accounts/<actname>/users/<un>.creds
 	// <op>/accounts/<actname>/users/<un>.nk
+	// new key layout is:
+	// keys/<op>/<first letter pk>/<second and third letters pk>/<pk>.nk
```

Comment:
I still think we should do away with this directory structure, overly complicates things IMO.

Reply:
I agree.